assets/dev/scss/frontend/widgets/spacer.scss

```diff
@@ -14,8 +14,8 @@
 
 	& > .elementor-widget-spacer {
 		width: var( --container-widget-width, var( --spacer-size ) );
-		align-self: stretch;
-		flex-shrink: 0;
+		--align-self: var( --container-widget-align-self, initial );
+		--flex-shrink: 0;
```

Comment:
I have limited the usage of `--align-self: stretch` to the 'container row`. The `container column` will use the `initial` value.